torch/csrc/inductor/aoti_runtime/utils_cuda.h

```diff
@@ -12,6 +12,9 @@
 #ifndef USE_ROCM
 #include <cuda_bf16.h>
 #include <cuda_fp16.h>
+#if defined(__CUDA_ARCH__) && (__CUDA_ARCH__ > 900)
```

Comment:
This file may not be always be compiled by a CUDA compiler, so this macro won't work. One way to solve this is to codegen "#include <cuda_fp8.h>" at the compile time by checking the arch version. Another way is to update cpp_builder.py to always use a nvcc compiler to compile wrapper.cpp.

Reply:
Also the actual cutlass kernels are still compiled by nvcc. The wrapper.cpp is compiled by g++.